src/StructuredLogger/BinaryLog.cs

```diff
@@ -27,7 +27,7 @@ public static IEnumerable<Record> ReadRecords(byte[] binlogBytes)
 
         public static Build ReadBuild(string filePath) => ReadBuild(filePath, progress: null);
 
-        public static Build ReadBuild(string filePath, Progress progress)
+        public static Build ReadBuild(string filePath, Progress progress, Progress bufferUsage = null)

```

Comment:
Oof, I hadn't realized that this is breaking public API.

Unfortunately I will have to back it out. Sorry about that.

Reply:
No problem.  Do revert.

Reply:
Good old overloads are still a thing.